### PR TITLE
[module] Controller for in the windtunnel to perform step inputs

### PR DESCRIPTION
**Pull request by fvantienen**
Simple module for controlling and logging windtunnel measurements for our hybrid plane.

### CODE REVIEW

File: sw/airborne/modules/ctrl/ctrl_windtunnel.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2015
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/ctrl_windtunnel.h:2]: fill name and date

File: sw/airborne/modules/ctrl/ctrl_windtunnel.c [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        // Finished
+        done = true;
+      }
+    }
+    else {
+      // By default increase flaps
+      ctrl_windtunnel_flaps.current += ctrl_windtunnel_flaps.step;
+
+      // Double the amount of steptime during double transition
+      if((ctrl_windtunnel_flaps.current == ctrl_windtunnel_flaps.min))
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/ctrl_windtunnel.c:118]: avoid equality comparison between float

File: sw/airborne/modules/ctrl/ctrl_windtunnel.c [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  ctrl_module_init();
+}
+
+void guidance_h_module_read_rc(void)
+{
+  // -MAX_PPRZ to MAX_PPRZ
+  ctrl_windtunnel.rc_throttle = radio_control.values[RADIO_THROTTLE];
+  ctrl_windtunnel.rc_roll = radio_control.values[RADIO_ROLL];
+  ctrl_windtunnel.rc_pitch = radio_control.values[RADIO_PITCH];
+  ctrl_windtunnel.rc_yaw = radio_control.values[RADIO_YAW];
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/ctrl_windtunnel.c:157]: only `rc_throttle` is used so far, should we really keep all channels ?

File: sw/airborne/modules/ctrl/ctrl_windtunnel.c (hunk 2 above)
Comment (dewagter) [reply sw/airborne/modules/ctrl/ctrl_windtunnel.c:118]: in this case, the ```min``` value is assigned to ```current``` by design so the comparison works.

File: sw/airborne/modules/ctrl/ctrl_windtunnel.c [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  int rc_throttle;
+  int rc_roll;
+  int rc_pitch;
+  int rc_yaw;
+  struct OrientationReps rotation;
+} ctrl_windtunnel;
+
+float ctrl_windtunnel_steptime = 8.0;
+struct min_max_ctrl_t ctrl_windtunnel_throttle = {.min = 0, .max = 0, .step = 1200};
+struct min_max_ctrl_t ctrl_windtunnel_flaps = {.min = -7200, .max = 0, .step = 2400};
```
[the remark this answers is not in the document]
Comment (dewagter) [reply sw/airborne/modules/ctrl/ctrl_windtunnel.c]: made values that can be automatically saved in the airframe file

File: sw/airborne/modules/ctrl/ctrl_windtunnel.c [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+void ctrl_module_init(void)
+{
+  ctrl_windtunnel.rc_throttle = 0;
+  ctrl_windtunnel.rc_roll = 0;
+  ctrl_windtunnel.rc_pitch = 0;
+  ctrl_windtunnel.rc_yaw = 0;
+
+  // Create a rotation to the windtunnel from the body
+  struct FloatEulers windtunnel_to_body_eulers =
+  {WINDTUNNEL_TO_BODY_PHI, WINDTUNNEL_TO_BODY_THETA, WINDTUNNEL_TO_BODY_PSI};
```
[the remark this answers is not in the document]
Comment (dewagter) [reply sw/airborne/modules/ctrl/ctrl_windtunnel.c:88]: done

File: sw/airborne/modules/ctrl/ctrl_windtunnel.h (hunk 1 above)
Comment (dewagter) [reply sw/airborne/modules/ctrl/ctrl_windtunnel.h]: done

**Pull request merged**